frontend/public/style/_overrides.scss

```diff
@@ -273,6 +273,11 @@ h6 {
   }
 }
 
+//set pf datalist item background to transparent
```

Comment:
Nit.  We try to alphabetize rules by selector.  So this should move to line 210.